README.md

```diff
@@ -31,7 +31,7 @@ const audience: Audience = {
 
 export const example = async () => {
   const streamdal = new Streamdal(config);
-  const result = await streamdal.processPipeline({
+  const result = await streamdal.process({
```

Comment:
`processPipeline` is no longer accurate in the multiple pipeline world. Once this is merged I'll update the docs and examples.